### ISSUE TITLE
check output from rest calls

### ISSUE THREAD

**Issue by JensTimmerman** [hpcugent/vsc-administration#5]
e.g. on https://github.com/hpcugent/vsc-administration/blob/master/lib/vsc/administration/user.py#L67

you could check the return code of the http call, like 404 or 503 or empty page or empty dict will all give serious errors here.

**Comment by itkovian** [comment]
This will get you to the except block. Which means this user will not be processed, which is correct afaik.

**Comment by JensTimmerman** [comment]
No, this will return a valid http response, with a return code, with json data indicating a part of the error, you then do double star magic on this json to create an account, witch fails with some invalid argument errors.

**Comment by JensTimmerman** [comment]
You should always validate the input you get here, and not just simply pass the result of an api call to a constructor, who know's what's in there.
Always Validate input!
